torch/jit/_script.py

```diff
@@ -944,7 +944,8 @@ def fail(self, *args, **kwargs):
         return fail
 
     for name, method in _get_methods(torch.nn.Module):
-        if name.startswith("__"):
+        print(name)
+        if name.startswith("__") or name.endswith("_call_impl"):
```

Comment:
this fixes the jit.load test locally but let's see if it breaks anything, waiting for CI to pass - I know I missed a print statement XD